src/modules/lyrics/lyrics.js

```diff
@@ -301,3 +250,50 @@ BetterLyrics.Lyrics = {
     return lyricsArray;
   },
 };
+
+document.addEventListener("blyrics-send-player-time", function(event) {
```

Comment:
Lets move this to ui/dom.js and call it from index.js

Reply:
I move the event listener to the `init` function in `index.js` and the `tickLyrics` function to `dom.js`